src/main/scala/mesosphere/marathon/api/akkahttp/AkkaHttpModule.scala

```diff
@@ -36,17 +39,23 @@ class AkkaHttpModule(conf: MarathonConf with HttpConf) extends AbstractModule {
     marathonSchedulerService: MarathonSchedulerService,
     appTasksRes: mesosphere.marathon.api.v2.AppTasksResource)(implicit
     actorSystem: ActorSystem,
+    materializer: Materializer,
     authenticator: Authenticator,
     authorizer: Authorizer,
-    electionService: ElectionService): AkkaHttpMarathonService = {
+    electionService: ElectionService,
```

Comment:
I thought we would get rid off Guice or is this the next step?

Reply:
It's a next step. Guice is moved to the upper level atm